# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1293540154 18000
#      Tue Dec 28 07:42:34 2010 -0500
# Node ID a140991387fb4aeb62b079297f7ac4cb0d783ce8
# Parent  1a7b41db5432559f4af4b2ac0232cd8b2176f664
legend.m: Add demo to legend for inline key.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-12-28  Ben Abbott <bpabbott@mac.com>
+
+	* plot/legend.m: Add demo to legend for inline key.
+
 2010-12-27  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/text.m: Add demo for text rotation and alignment.
 
 2010-12-26  Michael Godfrey <godfrey@isl.stanford.edu>
 
 	* specfun/legendre.m: Add @tex blocks to docstring.
 
diff --git a/scripts/plot/legend.m b/scripts/plot/legend.m
--- a/scripts/plot/legend.m
+++ b/scripts/plot/legend.m
@@ -979,16 +979,21 @@ function [hplots, text_strings] = getleg
       endif
       if (--k == 0)
         break;
       endif
     endif
   endwhile
 endfunction
 
+%!demo
+%! clf
+%! x = 0:1;
+%! plot (x, x, ";1;", x, 2*x, x, 3*x, ";3;")
+%! title ("1 is Blue, and 3 is Red")
 
 %!demo
 %! clf
 %! plot(1:10, 1:10, 1:10, fliplr(1:10));
 %! title("incline is blue and decline is green");
 %! legend({"I'm blue", "I'm green"}, "location", "east");
 
 %!demo
